operator-pipeline-images/operatorcert/catalog/package.py

```diff
@@ -0,0 +1,73 @@
+"""A module representing a package in the catalog."""
+
+from typing import Any
+
+
+class CatalogPackage:
+    """
+    A class representing a package in the catalog.
+    """
+
+    def __init__(self, definition: dict[str, Any], catalog: Any) -> None:
+        self.definition = definition
+        self.catalog = catalog
+
+    @property
+    def name(self) -> str:
+        """
+        A package name property.
+
+        Returns:
+            str: A name of a package.
+        """
+        return self.definition.get("name", "") or ""
+
+    def __str__(self) -> str:
+        """
+        String representation of a package given its name.
+        """
+        return self.name
+
+    def __repr__(self) -> str:
+        """
+        A representation of a CatalogPackage object.
+        """
+        return f"CatalogPackage({self.name})"
+
+    def get_channels(self) -> list[Any]:
+        """
+        Get all channels for a package.
+
+        Returns:
+            list: A list of all CatalogChannel objects for a package.
+        """
+        channels = []
+        for channel in self.catalog.get_all_channels():
+            if channel.package_name == self.name:
+                channels.append(channel)
+        return channels
+
+    def get_bundles(self) -> list[Any]:
```

Comment:
Similarly, this looks like it can just call `self.catalog.get_bundles_by_package()`

Reply:
Same here. There is a different condition in each method.